ucum-java8/src/main/java/systems/uom/ucum/UCUM.java

```diff
@@ -768,9 +786,116 @@ public static UCUM getInstance() {
+	//	public static final Unit OSMOLE = addUnit(MOLE);
+	//	public static final Unit PH = addUnit(ONE.multiply(-1).multiply(Math.log10(Quantities.getQuantity(1, MOLE).divide(Quantities.getQuantity(1, LITER)).getValue().doubleValue()))); is this okay?
+	//	public static final Unit GRAM_PERCENT = addUnit(GRAM.divide(DECI(LITER)));
+	//	public static final Unit SVEDBERG = addUnit(SECOND.multiply(1E-13));
+	//	public static final Unit HIGH_POWER_FIELD = addUnit(ONE);
+	//	public static final Unit LOW_POWER_FIELD = addUnit(ONE);
+	//	public static final Unit KATAL = addUnit(MOLE.divide(SECOND));
+	//	public static final Unit UNIT = addUnit(MICRO(MOLE).divide(MINUTE));
+
+	//	public static final Unit INTERNATIONAL_UNIT = UNDEFINED;
+	//	public static final Unit ARBITRARY_UNIT = UNDEFINED;
+	//	public static final Unit US_PHARMACOPEIA = UNDEFINED;
+	//	public static final Unit GPL = UNDEFINED;
+	//	public static final Unit MPL = UNDEFINED;
+	//	public static final Unit APL = UNDEFINED;
+	//	public static final Unit BETHESDA = UNDEFINED;
+	//	public static final Unit ANTI_FACTOR_XA = UNDEFINED;
+	//	public static final Unit TODD = UNDEFINED;
+	//	public static final Unit DYE = UNDEFINED;
+	//	public static final Unit SOMOGYI = UNDEFINED;
+	//	public static final Unit BODANSKY = UNDEFINED;
+	//	public static final Unit KING_ARMSTRONG = UNDEFINED;
+	//	public static final Unit KUNKEL = UNDEFINED;
+	//	public static final Unit MAC_LAGAN = UNDEFINED;
+	//	public static final Unit TUBERCULIN = UNDEFINED;
+	//	public static final Unit CELL_CULTURE_INFECTIOUS_50_PERCENT_DOSE = UNDEFINED;
+	//	public static final Unit TISSUE_CULTURE_INFECTIOUS_50_PERCENT_DOSE = UNDEFINED;
+	//	public static final Unit EMBRYO_CULTURE_INFECTIOUS_50_PERCENT_DOSE = UNDEFINED;
+	//	public static final Unit PLAQUE_FORMING = UNDEFINED;
+	//	public static final Unit FOCUS_FORMING = UNDEFINED;
+	//	public static final Unit COLONY_FORMING = UNDEFINED;
+	//	public static final Unit INDEX_OF_REACTIVITY = UNDEFINED;
+	//	public static final Unit BIOEQUIVALENT_ALLERGEN = UNDEFINED;
+	//	public static final Unit ALLERGEN = UNDEFINED;
+	//	public static final Unit ALLERGEN_FOR_AMBROSIA_ARTEMISIIFOLIA = UNDEFINED;
+	//	public static final Unit PROTEIN_NITROGEN = UNDEFINED;
+	//	public static final Unit LIMIT_OF_FLOCCULATION = UNDEFINED;
+	//	public static final Unit D_ANTIGEN = UNDEFINED;
+	//	public static final Unit FIBRINOGEN_EQUIVALENT = UNDEFINED;
+	//	public static final Unit ELISA = UNDEFINED;
+	//	public static final Unit EHRLICH = UNDEFINED;
+	//	public static final Unit CHEMICAL = UNDEFINED;
+
+	// ///////////////////////////////
+	// LEVELS UNITS: UCUM 4.5 §45 //
+	// //////////////////////////////
+	//	public static final Unit<Level<?>> NEPER = addUnit(new ProductUnit<Level<?>>(ONE.multiply(Math.log(1))));
+	//	public static final Unit<Level<?>> BEL = addUnit(new ProductUnit<Level<?>>(ONE.multiply(Math.log10(1))));
+
+	//	public static final Unit<Level<Pressure>> BEL_SOUND = addUnit(
```

Comment:
I'll start saying that this block is wrong, but I didn't know exactly how to make them, if I just use values, quantities (as some of them is in a weird unit as this one `2lg(2 10*-5.Pa)`), and they can't be produced because I can't cast a TransformedUnit in a ProductUnit, I'll see it later.